src/Sylius/Component/Core/Customer/Statistics/PerChannelCustomerStatistics.php

```diff
@@ -41,6 +41,9 @@ public function getChannel(): ChannelInterface
 
     public function getAverageOrderValue(): int
     {
+        if (0 == $this->ordersCount) {
+            return 0;
+        }
         return (int) round($this->ordersValue / $this->ordersCount);
```

Comment:
Per coding style, there should be empty line before `return`.
```suggestion
        }
        
        return (int) round($this->ordersValue / $this->ordersCount);
```